### PR TITLE
Cleanup of #5187

### PR DESCRIPTION
**Pull request by bmarty**
See #5187 for more details.

Will squash and merge to cleanup git history.

After my change:

image

Fixes https://github.com/vector-im/element-android/issues/5114

### CODE REVIEW

File: vector/src/main/java/im/vector/app/features/widgets/permissions/RoomWidgetPermissionBottomSheet.kt [hunk 1]
```diff
@@ -102,8 +102,9 @@ class RoomWidgetPermissionBottomSheet :
     }
 
     override fun onCancel(dialog: DialogInterface) {
+        viewModel.handle(RoomWidgetPermissionActions.BlockWidget)
```
Comment (mnaturel) [inline vector/src/main/java/im/vector/app/features/widgets/permissions/RoomWidgetPermissionBottomSheet.kt:105]: Maybe should we reuse the existing `doDecline` method?

**Review by mnaturel — APPROVED** [review]
I have just added a small remark to mutualize some code.

File: vector/src/main/java/im/vector/app/features/widgets/permissions/RoomWidgetPermissionBottomSheet.kt (hunk 1 above)
Comment (bmarty) [reply vector/src/main/java/im/vector/app/features/widgets/permissions/RoomWidgetPermissionBottomSheet.kt:105]: https://github.com/vector-im/element-android/pull/5187#discussion_r813882777 😄

Comment (mnaturel) [reply vector/src/main/java/im/vector/app/features/widgets/permissions/RoomWidgetPermissionBottomSheet.kt:105]: I see. It is up to you but we could create a sub method without including call to `dismiss()` as you suggested?

**Review by Claire1817 — APPROVED** [review]

**Pull request merged**